indico/modules/events/__init__.py

```diff
@@ -245,3 +245,9 @@ def _event_cloned(old_event, new_event, **kwargs):
     new_event.contact_title = old_event.contact_title
     new_event.contact_emails = old_event.contact_emails
     new_event.contact_phones = old_event.contact_phones
+
+
+@signals.event.moved.connect
+def _event_request_moved(event, old_parent):
+    if event.pending_move_request and event.pending_move_request.category == event.category:
```

Comment:
This queries the request twice since it's a normal property...

Either make it a method so it's clear that it should be assigned and not just used many times, or maybe turn it into a relationship with a custom primaryjoin criterion that includes the state and has `viewonly=True`.